### PR TITLE
BRD/DRG/MCH/MNK/NIN/SAM Self-Healing all updated.

### PR DESCRIPTION
**Pull request by Daemeous**
Updated and tested all jobs (except BRD which I don't have yet), lowered Nin priority quite a bit to avoid CD drift.

### CODE REVIEW

File: XIVSlothCombo/Combos/PvE/DRG.cs [hunk 1]
```diff
@@ -177,6 +181,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                                 if (WasLastWeaponskill(HeavensThrust) && GetRemainingCharges(SpineshatterDive) > 0 && !WasLastAction(SpineshatterDive) && openerOptions is 0 or 1 or 2)
                                     return SpineshatterDive;
+
+                                // healing - please move if not appropriate priority
```
Comment (damolitionn) [inline XIVSlothCombo/Combos/PvE/DRG.cs:185]: Why are these in the opener

File: XIVSlothCombo/Combos/PvE/SAM.cs [hunk 2]
```diff
@@ -219,6 +223,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                                 if (gauge.Kenki >= 25 && IsOnCooldown(Shoha))
                                     return Shinten;
+
+                                // healing - please move if not appropriate this high priority
```
Comment (damolitionn) [inline XIVSlothCombo/Combos/PvE/SAM.cs:227]: Same with these

File: XIVSlothCombo/Combos/PvE/SAM.cs [hunk 3]
```diff
@@ -482,6 +495,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                                     if (IsEnabled(CustomComboPreset.SAM_ST_GekkoCombo_CDs_Shoha) && Shoha.LevelChecked() && gauge.MeditationStacks == 3)
                                         return Shoha;
+
+                                    // healing - please move if not appropriate this high priority
```
Comment (damolitionn) [inline XIVSlothCombo/Combos/PvE/SAM.cs:499]: These are nested under `SAM_ST_GekkoCombo_CDs` which means your customcombopreset will need to be adjusted accordingly

Comment (Daemeous) [reply XIVSlothCombo/Combos/PvE/SAM.cs:499]: Do you think that's the best way to do it? I was fretting over doing that, which would make it unique among all the implementations or having it's own separate canWeave location for it. But I must confess, the Sam code is a bit too big brain for me to figure out quite where that would be

Comment (damolitionn) [reply XIVSlothCombo/Combos/PvE/SAM.cs:499]: It's either that or move it somewhere else. Users will need to have `SAM_ST_GekkoCombo_CDs` on as well then and if it's not nested then there's no indication that it's required to be on for your healing feature to work.

**Pull request merged**